dev/FixtureBlueprint.php

```diff
@@ -183,14 +183,8 @@ public function createObject($identifier, $data = null, $fixtures = null) {
 
 			// If LastEdited was set in the fixture, set it here
 			if($data && array_key_exists('LastEdited', $data)) {
-				$edited = $this->parseValue($data['LastEdited'], $fixtures);
```

Comment:
Or if you want to do it generically `$class = ClassInfo::table_for_object_field($class, 'LastEdited');`